ember-server/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -169,20 +170,31 @@ private[server] object ServerHelpers {
   private[internal] def postProcessResponse[F[_]: Timer: Monad](
       req: Request[F],
       resp: Response[F]): F[Response[F]] = {
-    val reqHasClose = req.headers.exists {
-      // We know this is raw because we have not parsed any headers in the underlying alg.
-      // If Headers are being parsed into processed for in ParseHeaders this is incorrect.
-      case Header.Raw(name, values) => name == connectionCi && values.contains(closeCi.value)
```

Comment:
`toLowerCase` was missing here on a raw header! These means that if a client made a request with `Connection: Keep-Alive` (as opposed to `keep-alive`), the server would've closed the connection!